### DISCUSSION TITLE
At the "invite" page, make the URL that is shown at "Powered by ..." configurable.

### DISCUSSION THREAD

**Discussion by colfes** [lukevella/rallly#921] [Ideas]
Currently, on the "invite" page (where the users fill out the Doodle), the footer says "Powered by rallly.co" - which isn't really true for self-hosted versions. What worsens it, is that the German Translation for that is "Betrieben von" which translates back to "Run by", and then it's entirely untrue.

If would be great to have a way to change this footer to something configurable in the .env, or at least at build time.
Then, one could also "misuse" this footer to link an Imprint, for countries in which an imprint can be mandatory (e.g. Germany, under certain circumstances no one understands, so you should just use one)

**Comment by mowtim** [comment]
I'm new to pretty much everything that has to do with git, docker and, of course, rallly - just so you know what to expect if i answer.
In the repo could you just go into apps/web/public/locales/de/app.json and change line 104 to your liking?
Wouldn't work if you use the docker image, i guess.
